QueryBuilder/Values/NullifValue.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SqlKata.Values {
+    public class NullifValue : AbstractFunctionValue {
+
+        public object Expression => Arguments[0];
+
+        public object Condition => Arguments[1];
```

Comment:
this is not a condition... it's a second expression actually.
if it equals to first, then the first value is returned. Let's change its name to Operand or ReferenceValue or something like that